chapter_01.md

```diff
@@ -1,2 +1,550 @@
+
+Both, Bounded Contexts and Context Maps are great conceptual tools to understand how to structure your teams and your software, but more importantly, these concepts help you to focus on business value. 
+
+
+### Focus on Business Value
+
+When working on real applications, you need to focus on what is going to bring more value and solve more problems for your business. As a very practical person, I tend to analyze use cases and how Bounded Context in conjunction with Context Maps provides a business flow end to end. 
+
+For this particular example, the conference scenario, we will be looking at the Call for Proposals basic flow, which looks like this:
+Potential Speaker Submits a proposal via the conference website
+Board/Committee review the proposal
+If the proposal gets accepted
+A new agenda item is created and published into the Agenda Page
+A notification is sent via email for Accepted and Rejected proposals
+
+You need to pay attention to your human/people interactions as these interactions tend to require asynchronous behaviors such as reminders, notifications, alerts as well as User Interfaces which will need to be carefully designed. As engineers, we tend to oversimplify and underestimate the amount of work and iterations that crafting a good user experience might take.
+
+The user interfaces for covering this simple scenario look like this: 
+
+![chapter_01_08](images/chapter_01_08.png)
+
+The main page inside the Conference Site displays the Agenda divided by days. The items inside the agenda are the ones that are already confirmed and were approved by the committee.
+
+The main page also allow potential speakers to submit proposals by filling up a form:
+
+![chapter_01_09](images/chapter_01_09.png)
+
+Once the proposal is submitted the potential speaker will need to wait for Approval or Rejection by the committee. 
+
+The committee members have a back-office page where they can Approve or Reject each submitted proposal: 
+
+![chapter_01_10](images/chapter_01_10.png)
+
+The back-office page also offers Board members the option to send email notifications to the potential speakers.
+
+![chapter_01_11](images/chapter_01_11.png)
+
+Once again, you can notice the simplification of this scenario on purpose to establish a base set of functionality, quickly iterate and get it working and then expand the requirements. 
+
+#### Architecture and Services 
+
+From an architectural perspective it might look more like this: 
+![chapter_01_12](images/chapter_01_12.png)
+
+Where the  User Interface with some routing capability is required to forward requests to the Call for Proposals Service (C4P) to the Agenda or Emails Service. 
+
+All the communications happen via HTTP/Rest invocations. 
+
+### API Gateway / User Interface
```

Comment:
I'm afraid we might have given too much emphasis to the UI of the demo application and not enough on the UI and API Gateway importance when related to DDD concept...

Reply:
I do agree that the UI might not be relevant, but once again, The gateway comes as result of having a bunch of services instead of a single monolith. I can drill down on why and for example, the hard requirement on SSO, but mapping that to DDD at the conceptual level feels like quite a stretch, as DDD doesn't go into actual implementation details for that exact same reason. Having a gateway is just a consequence of implementing DDD with a bunch of microservices that are implemented in different containers. @kmacedovarela does it make sense? I am happy to adapt the content to more accurately reflect that.